tika-bundle/pom.xml

```diff
@@ -21,7 +21,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.codice.thirdparty</groupId>
     <artifactId>tika-bundle</artifactId>
-    <version>1.18.0_2</version>
+    <version>1.19</version>
```

Comment:
Shouldn't this be `1.19.0_1` to be consistent with previous versions?

Reply:
👍 Good catch